providers/curation/github.js

```diff
@@ -93,14 +93,23 @@ class GitHubCurationService {
         )
       )
     )
-    return (userGithub || serviceGithub).pullRequests.create({
+    const retVal = await (userGithub || serviceGithub).pullRequests.create({
```

Comment:
general naming comment. we tend to use whole words and semantic naming. So `retVal` => `result` and `commentObj` => `comment`

Reply:
Naming updated.